src/module_wrap.cc

```diff
@@ -502,6 +502,44 @@ void ModuleWrap::Resolve(const FunctionCallbackInfo<Value>& args) {
   args.GetReturnValue().Set(result.ToObject(env));
 }
 
+static MaybeLocal<Promise> ImportModuleDynamically(Local<Context> context,
+                                                   Local<String> referrer,
+                                                   Local<String> specifier) {
+  Isolate* iso = context->GetIsolate();
+  // TODO(jkrems): ensure proper context is used
+  Environment* env = Environment::GetCurrent(context);
+  HandleScope handle_scope(iso);
+
+  Local<Function> import_callback =
```

Comment:
There's currently an unhandled error case when dynamic import is used in an environment that doesn't have the callback defined (yet).